removing references to Macro and Partial view macro (#5979)
umbraco/UmbracoDocs: 13/umbraco-cms/reference/templating/macros/README.md

_Describes how to set up a macro, use macro parameters & configuring caching. Defines the different types of macros and provides details on the different macro engine APIs and their usage_

{% hint style="warning" %}

> Macros and Partial View Macros are deprecated and will be removed in the next version. We recommend using Partial Views or Blocks in Rich Text Editor.

Author: ```suggestion
Macros and Partial View Macros will be removed in the next version. Consider using Partial Views or Blocks in Rich Text Editor.
```
